### PR TITLE
Exercicies for conditionals, loops and whiles done!

### PR DESCRIPTION
**Pull request by nilete22**

### CODE REVIEW

File: challenge-03_loops-conditionals/loops-conditionals/conditionals/exercise_00.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+//1. At the start of the program print out: "--- Animal Checker Program ---"
+console.log("--- Animal Checker Program ---");
+//2. Create a variable called animal that starts at the value "cat"
+var animal = "dog";
+//3. Create an if statement that checks if animal is equal to "cat"
+//4. If it is equal to "cat", print out "Meow!" to the screen
+//5. If it is equal to "dog", print out "Woof!" to the screen
+//6. If it is neither "cat" nor "dog", print out "Must be an alien"
+if (animal === "cat") {
```
Comment (tictools) [inline challenge-03_loops-conditionals/loops-conditionals/conditionals/exercise_00.js]: ⚠️ FIX: Change animal to different values to make sure it works in all cases

File: challenge-03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    The program will return the variable is even or odd.
+
+    Change the variable to test it works for different cases.
+    Print result.
+
+    HINT: modulus operator ( % ) is your friend
+*/
+let number = 13;
+
+if (number % 2 === 0) {
```
Comment (tictools) [inline challenge-03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js]: ⚠️ FIX: Change the variable to test it works for different cases. Print result.

File: challenge-03_loops-conditionals/loops-conditionals/loops/exercise_02.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    Checkpoint! 40
+    Half way there!
+    Checkpoint! 60
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+for (let i = 1; i <= 100; i++) {
```
Comment (tictools) [inline challenge-03_loops-conditionals/loops-conditionals/loops/exercise_02.js]: This is the output of your script...
```
Checkpoint! 10
Checkpoint! 20
Checkpoint! 30
Checkpoint! 40
Checkpoint! 50
Checkpoint! 60
Checkpoint! 70
Checkpoint! 80
Checkpoint! 90
Checkpoint! 100
All, done!
```

...have you tested it before pushing? 🤔

⚠️ FIX: needs work.

File: challenge-03_loops-conditionals/loops-conditionals/while/exercise_01.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+//1. Create a while loop that will print out all the even numbers between 10 and 40.
+console.log("Even numbers between 10 and 40:");
+let evenNumber = 10;
+while (evenNumber <= 40) {
```
Comment (tictools) [inline challenge-03_loops-conditionals/loops-conditionals/while/exercise_01.js]: These loops should iterate over all numbers in between 10 and 40, printing only the odd/even ones, depending on its own logic.

⚠️ FIX: iterate over all numbers and print only the odd/even ones

File: challenge-03_loops-conditionals/loops-conditionals/while/exercise_02.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    Checkpoint! 40
+    Half way there!
+    Checkpoint! 60
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+let i = 1;
```
Comment (tictools) [inline challenge-03_loops-conditionals/loops-conditionals/while/exercise_02.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case?

⚠️ FIX: rename properly `i` variable

This is the output of your script...
```
Checkpoint! 10
Checkpoint! 20
Checkpoint! 30
Checkpoint! 40
Checkpoint! 50
Checkpoint! 60
Checkpoint! 70
Checkpoint! 80
Checkpoint! 90
Checkpoint! 100
All, done!
```

...have you tested it before pushing? 🤔

⚠️ FIX: needs work.

File: challenge-03_loops-conditionals/loops-conditionals/conditionals/exercise_02.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  if (age < 3) {
+    console.log("You're just a baby!");
+  } else if (age < 13) {
+    console.log("You are in elementary school, kid.");
+  } else if (age < 18) {
+    console.log("Nice, you're a teenager!");
+  } else {
+    console.log("Most places consider you an adult.");
+  }
+*/
```
Comment (tictools) [inline challenge-03_loops-conditionals/loops-conditionals/conditionals/exercise_02.js]: FIX: format code.

**Review by tictools — CHANGES_REQUESTED** [review]

File: .githubworkflows/Pipeline.yaml
Comment (tictools) [inline .githubworkflows/Pipeline.yaml:1]: Actions path is `.github/workflows/Pipeline.yaml`, not `.githubworkflows/Pipeline.yaml`.

No action will be executed until you fix this path.

FIX: write correct path

**Review by tictools — CHANGES_REQUESTED** [review]

**Review by tictools — APPROVED** [review]

**Review by Lye-Frozen-Heart — APPROVED** [review]

**Pull request merged**